build/build.rb

```diff
@@ -89,6 +90,8 @@ def verify_run(output)
     file_name = "#{name}-#{target}-#{build_label}-#{timestamp}-#{git_checkout.commit_sha}.#{FILE_TYPE[imgfac_target]}"
     destination = DESTINATION_DIRECTORY.join(file_name)
     $log.info `mv  #{source} #{destination}`
+    $log.info `cd #{DESTINATION_DIRECTORY}/; /usr/bin/md5sum * >> md5checksums\""`
+    $log.info `cd #{DESTINATION_DIRECTORY}/; /usr/bin/sha256sum * >> 256checksums\""`
```

Comment:
In a future iteration, you could change from shelling out to using [Ruby's Digest class] (http://ruby-doc.org/stdlib-2.1.0/libdoc/digest/rdoc/Digest.html)